docs/data-model.md

```diff
@@ -28,6 +28,8 @@ erDiagram
   School {
     uuid id PK
     string urn FK "Primary key for schools in GIAS"
+    bool placements_enabled "Indicates if the School has been onboarded into the School Placements service"
```

Comment:
Rather than a `boolean` should these be `datetime` then we can also track when the onboarding happened.

Reply:
This could be useful, and it's a convenient way to record when a school was onboarded. But I don't necessarily want to make any assumptions about what we should track yet, nor how exactly to track it.

For example, there may be a need for an audit trail which shows the date/time a school was onboarded but also who did it. Or perhaps it's possible to 'offboard' a school from the service temporarily, and we'd need the audit trail to show a complete onboard/offboard history for schools. In that case, we would probably model the data slightly differently.

I'm also conscious that some of these needs may also potentially be met by the [DfE Analytics gem](https://github.com/DfE-Digital/dfe-analytics) and platform. The approach is to send granular event data to BigQuery for later filtering and analysis – so I could imagine us being able to source the date/time a school was enabled via that too.

So for that reason, I'd probably be keen to stick with a `boolean` field. That seems to be the simplest thing we can do to meet the immediate need of our service.

Reply:
Na. I'm happy to concede!